### DISCUSSION TITLE
Changing home screen notifications to "read" and changin top nav bar buttons?

### DISCUSSION THREAD

**Discussion by wafflecrate** [inventree/InvenTree#6020] [Q&A] [answered]
I've been exploring the Inventree "home" menu and wanted to know if two things are possible:

1. As you can see in my attached image my home page has notifications set for "Latest Parts - 68, Recently Updated - 94". The blue numbers keep going up and even when I click into those menus and explore the entry it never sets them to "read". How do I set these to "read" so I don't have an ever-rising blue highlight number? I just want to click into them, read the update, and when I exit out have it reset back to 0.
2. The top navigation bar says: Parts - Stock - Build - Buy - Sell. It'd like to remove the "Sell" button. Is it possible to delete or even re-arrange the top nav buttons?
![Home screen](https://github.com/inventree/InvenTree/assets/67816847/b1e715f1-2ec5-4a44-9c95-97303949ac20)

**Comment by wafflecrate** [comment]
on a similar note- is it possible to permanently rearrange and remove the categories that appear in the "Part Stock" section? Part, IPN, Description, Stock, etc. I know that you can uncheck and have each one hidden from view, but they all return to default once you exit the page. Part, IPN
![table](https://github.com/inventree/InvenTree/assets/67816847/d9e6c6dd-d7a2-45c5-a224-70903507a58f)

**Reply by SchrodingersGat** [reply]
No, it is not possible to rearrange the columns in these tables. We are currently refactoring the entire front-end interface to use React, perhaps that will be technically possible then? This would not be a priority for the development team though.

**Reply by wafflecrate** [reply]
Sure, and thanks for confirming. Will continue to follow throughout the next releases.

**Comment by wolflu05** [comment]
You should be able to disable the sell functionality for a user by removing the sell permission from the users group.

For the table columns, rearranging is not possible, but this could be a FR for the new Platform UI. The current ui does not longer revive active feature development. Saving this to the local storage is also a good idea.

**Reply by wafflecrate** [reply]
I removed the Sell permissions from the users group but the "Sell" button is still in the top nav bar.

**Reply by SchrodingersGat** [reply]
This is not currently possible, but perhaps can be addressed in the new user interface

**Reply by wolflu05** [reply]
oh, sorry, thought that this would be possible, because the `engineer` account in the demo dataset has not all menu items, but it appears that Sell cannot be removed.

**Comment by SchrodingersGat** [comment, marked as answer]
Regarding question 1: The home page / dashboard items are not "readable" - they provide a list of items which have been most recently added or updated. There is no way to instruct the system that these have been "read" or "observed".
